cognite/client/_api/files.py

```diff
@@ -52,19 +56,22 @@ def __call__(
         Yields:
             Union[FileMetadata, FileMetadataList]: yields FileMetadata one by one if chunk is not specified, else FileMetadataList objects.
         """
+        if (root_asset_ids and not isinstance(root_asset_ids[0], dict)) or root_asset_external_ids:
```

Comment:
Don't need the backwards compatiblity on root_asset_ids being `List[Dict]` here, since we never supported it.